gdk4-macos/Cargo.toml

```diff
@@ -15,13 +15,15 @@ version.workspace = true
 
 [features]
 v4_8 = ["gdk4-macos-sys/v4_8"]
+cocoa = ["dep:cocoa"]
```

Comment:
I think this is unnecessary but doesn't hurt for making it more explicit (every optional dependency is automatically also a feature)

Reply:
I prefer them being explicit personally :)

Reply:
Same